Cargo.toml

```diff
@@ -33,13 +33,17 @@ cblas-sys = { version = "0.1.4", default-features = false, optional = true }
 libc = { version = "0.2", default-features = false, optional = true }
 cudarc = { version = "0.7.2", default-features = false, optional = true }
 num-traits = { version = "0.2.15", default-features = false }
+# safetensors = { version = "0.2", default-features = false, optional = true }
+safetensors = { git = "https://github.com/huggingface/safetensors", default-features = false, optional = true }
```

Comment:
Was this a holdover from developing? Which should be used?

Reply:
I need to release `0.3` . Should happen this week. (I will release python release candidate first, there has been a relatively big change in how things are saved, everythng backward compatible, but files on disk will be different now, to get alignement properly done).